dataflow/gpu-workers/e2e_test.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+import uuid
+
+from google.cloud import storage
+import pytest
+
+SUFFIX = uuid.uuid4().hex[0:6]
+PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
+BUCKET_NAME = f"dataflow-gpu-test-{SUFFIX}"
+IMAGE_NAME = f"gcr.io/{PROJECT}/dataflow/gpu-workers/test-{SUFFIX}:latest"
+
+
+@pytest.fixture(scope="session")
+def bucket_name() -> str:
+    storage_client = storage.Client()
+    bucket = storage_client.create_bucket(BUCKET_NAME)
+
+    yield BUCKET_NAME
+
+    bucket.delete()
```

Comment:
```suggestion
    bucket.delete(force=True)
```

This will let you delete the bucket even if it's not empty